metadata-ingestion/src/datahub/ingestion/graph/client.py

```diff
@@ -171,6 +171,21 @@ def test_connection(self) -> None:
             self.server_id = _MISSING_SERVER_ID
             logger.debug(f"Failed to get server id due to {e}")
 
+    @property
+    def frontend_base_url(self) -> str:
+        """Get the public-facing base url of the frontend
+
+        This url can be used to construct links to the frontend. The url will not include a trailing slash.
+        """
+
+        if not self.server_config:
+            self.test_connection()
+
+        base_url = self.server_config.get("baseUrl")
+        if not base_url:
+            raise ValueError("baseUrl not found in server config")
```

Comment:
add a note that this only works for acryl cloud

Reply:
Ah, didn't realize. You think that's preferred over not including altogether?

Reply:
yeah I think so